system_metrics_collector/src/moving_average_statistics/types.hpp

```diff
@@ -15,12 +15,26 @@
 #ifndef MOVING_AVERAGE_STATISTICS__TYPES_HPP_
 #define MOVING_AVERAGE_STATISTICS__TYPES_HPP_
 
+#include <array>
 #include <sstream>
 #include <string>
 
+#include "metrics_statistics_msgs/msg/statistic_data_type.hpp"
+
 namespace moving_average_statistics
 {
 
+/**
+ *  Enumeration of the various StatisticDataTypes
+ */
+constexpr const std::array<uint8_t, 5> STATISTICS_DATA_TYPES = {
+  metrics_statistics_msgs::msg::StatisticDataType::STATISTICS_DATA_TYPE_AVERAGE,
+  metrics_statistics_msgs::msg::StatisticDataType::STATISTICS_DATA_TYPE_MINIMUM,
+  metrics_statistics_msgs::msg::StatisticDataType::STATISTICS_DATA_TYPE_MAXIMUM,
+  metrics_statistics_msgs::msg::StatisticDataType::STATISTICS_DATA_TYPE_STDDEV,
+  metrics_statistics_msgs::msg::StatisticDataType::STATISTICS_DATA_TYPE_SAMPLE_COUNT
```

Comment:
alphasort

Reply:
This array variable is no longer part of this PR.